Pendulum/Views/Subviews/CDEventCell.swift

```diff
@@ -101,6 +101,7 @@ struct EventCell: View {
                 }
             }
             .buttonStyle(.plain)
+            .animation(.default, value: penpal)
```

Comment:
We''re not very consistent in where we apply animation and where not I believe.

Reply:
These are mostly an attempt to try and get background synced changes to animate in: so far, unsuccessfully.